src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println(this.getClass().getSimpleName() + " started work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(this.getClass().getSimpleName() + " stopped working");
```

Comment:
Similarly, the stop message should be more informative about the `Excavator`'s state. Mentioning the specific action that has been stopped could improve the clarity of the message.